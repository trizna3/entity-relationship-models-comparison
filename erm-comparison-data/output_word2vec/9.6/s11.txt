persons-person;accounts
life insurance-life insurance
account-insurance
real estate types-real estate type
contract-contract
addresses;real estate insurance-real estate insurance
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-real estate type-real estate insurance
DECOMPOSE_ATTRIBUTE:S-address-real estate insurance
CONTRACT_11_ASSOCIATION:E-addresses-real estate insurance
CHANGE_CARDINALITY:S-accounts-person
CONTRACT_11_ASSOCIATION:S-person-accounts
