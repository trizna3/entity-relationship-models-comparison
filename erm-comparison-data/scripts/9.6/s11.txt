entity set person {
};

entity set accounts {
	"username"
	"password"
};

entity set contract {
	"data"
};

entity set insurance {
	"date from"
	"date to"
};


entity set "life insurance" {
	"amount"
	"for whom"
};

entity set "real estate insurance" {
	"real estate type"
	"amount"
	"address"
	"living area size"
	"basement size"
};

association between 
{
	person ONE 
	accounts MANY
}{};

association between 
{
	accounts ONE 
	contract MANY
}{};


association between 
{
	contract MANY
	insurance ONE
}{};

generalization from "real estate insurance" to insurance;

generalization from "life insurance" to insurance;
