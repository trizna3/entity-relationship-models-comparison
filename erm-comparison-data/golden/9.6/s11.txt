account-accounts
contract-contract
life insurance-life insurance
real estate insurance-real estate insurance
persons-person
addresses-address
real estate types-real estate type
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-real estate type-real estate insurance
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estate insurance